### ISSUE TITLE
[Keyboard Navigation – Howdy – Scripts]: Focus indicator is not visible on "Next, Clone, Edit and Delete" buttons present under “Script” tab.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#9]
**User Experience:**
Users who rely on keyboards will affect to navigate if focus indicator is not visible on multiple controls in the page.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in edge browser.
Step 2: Open this URL: https://hcl-szz1925.howdy.ai/dashboard in edge.
Step 3: “Dashboard” page gets opened.
Step 4: Navigate to “Scripts” tab and select it.
Step 5: Navigate all controls inside Scripts tab using keyboard tab key.
Step 6: Verify whether focus indicator is visible on "Next, Clone, Edit and Delete" buttons present under “Script” tab.

**Actual Result:**
Focus indicator is not visible on "Next, Clone, Edit and Delete" buttons present under “Script” tab.

**Expected Result:**
Focus indicator should be visible on "Next, Clone, Edit and Delete" buttons present under “Script” tab.

**Note**: Same issue repros on "Save and Delete" buttons which are present in "Edit and Clone" dialogs.
**Note 1** Same issue repro for "Howdy, Knowledge base, Howdy Basics, Powered by groove" link.
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Please refer the attachment:MAS2.4.7_Howdy for pms.wmv

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)
[MAS2.4.7_Howdy_Scripts tab.zip](https://github.com/howdyai/howdy-website/files/2973612/MAS2.4.7_Howdy_Scripts.tab.zip)
[MAS2.4.7_Howdy for pms.zip](https://github.com/howdyai/howdy-website/files/2973753/MAS2.4.7_Howdy.for.pms.zip)

**Comment by benbrown** [comment]
This has been corrected.

**Comment by v-gasikk** [comment]
Verified and Issue is Fixed.
Focus indicator is visible on  "Next, Clone, Edit, Delete and Save" buttons.

Note1: This page has been deprecated and removed from the site.- Closing this issue as per dev comments
